explain-layers/tiles.csv

```diff
@@ -1,5 +1,5 @@
 TileID,Title,Group,Description,Link
-0,Empty,,"Empty tile, used as an eraser in the map editor.",
+0,Empty,,"Empty Tile, used as an eraser in the Map Editor.",
```

Comment:
```suggestion
0,Empty,,"Empty tile, used as an eraser in the map editor.",
```
English doesn't capitalize nouns, only if they're in the beginning of a sentence or if they're proper nouns. This document seems to refer to tile names as proper nouns, that's why some nouns are capitalized, I think.

Reply:
Not a burden I want to put on a first-time contributor.

Reply:
Thank you, but if you tell me where I got it wrong, I can either change it accordingly, or learn from it at least.